core/space/services/services_sharing.go

```diff
@@ -226,11 +226,12 @@ func (s *Space) OpenSharedFile(ctx context.Context, hash, password, filename str
 	}, nil
 }
 
-func (s *Space) ShareFilesViaPublicKey(ctx context.Context, paths []domain.FullPath, pubkeys []crypto.PubKey) error {
+func (s *Space) ShareFilesViaPublicKey(ctx context.Context, paths *[]domain.FullPath, pubkeys []crypto.PubKey) error {
```

Comment:
Maybe add a comment on this function that `paths` would be modified. Personally, I think it would have been better if the function returned the modified `paths` instead of mutating the input value, but eitherways a comment should help make it more apparent.

Reply:
Fixed in [26a4678](https://github.com/FleekHQ/space-daemon/pull/161/commits/26a4678264f4dc0e08047c1693559ae8308a5772).